evap/settings.py

```diff
@@ -407,12 +407,10 @@ def CHARACTER_ALLOWED_IN_NAME(character):  # pylint: disable=invalid-name
 # Create a localsettings.py if you want to locally override settings
 # and don't want the changes to appear in 'git status'.
 try:
-    # if a localsettings file exists (vagrant), this will cause wildcard-import errors
-    # if it does not, (GitHub), it would cause useless-suppression
-    # pylint: disable=unused-wildcard-import,wildcard-import,useless-suppression
+    # localsettings file may exist (vagrant) or do not (GitHub)
```

Comment:
This comment may now have confusing grammar or do not.

```suggestion
    # localsettings file may (vagrant) or may not (CI run) exist
```